app/models/big_blue_button_conference.rb

```diff
@@ -117,11 +142,27 @@ def fetch_recordings
 
   def delete_recording(recording_id)
     response = send_request(:deleteRecordings, {
-      :recordID => recording_id,
```

Comment:
In Ruby I prefer to leave the terminal comma in a hash or array, because it reduces the super common bug of adding another line to a multiline array or hash and forgetting to add a comma to the previous line, and it makes diffs cleaner because you don't have to add an incidental change to the previous line.